tests/sc_inserts.test/runit

```diff
@@ -70,39 +70,52 @@ function run_test
 {
     typeset ipid=''
 
-    cdb2sql ${CDB2_OPTIONS} $dbnm default "truncate $tbl"
+    cdb2sql ${CDB2_OPTIONS} ${DBNAME} default "drop table if exists $tbl"
+    cdb2sql ${CDB2_OPTIONS} ${DBNAME} default "create table $tbl {
+schema
+{
+    int      a
+    cstring  b[32]
+    blob     c
+    int      d
+    int      e
+    int      f
+}
+}
+"
 
-    insert_records 0 &
+    insert_records $nrecs 0.01 &
     typeset ipid=$!
 
-    do_rebuild_track_pid $dbnm $tbl $ipid
+    cdb2sql ${CDB2_OPTIONS} ${DBNAME} default "alter table $tbl { `cat $tbl.csc2 ` }"
+    do_verify $tbl
+
+    do_rebuild_track_pid ${DBNAME} $tbl $ipid
     wait
 
     assertcnt $tbl $nrecs
     do_verify $tbl
 }
 
 
-
-
 echo "running test in machine $(hostname):${PWD}"
 
-cdb2sql ${CDB2_OPTIONS} $dbnm default "drop table $tbl"
-cdb2sql ${CDB2_OPTIONS} $dbnm default "create table $tbl  { `cat $tbl.csc2 ` }"
-
-
-master=`cdb2sql -tabs ${CDB2_OPTIONS} $dbnm default 'exec procedure sys.cmd.send("bdb cluster")' | grep MASTER | cut -f1 -d":" | tr -d '[:space:]'`
+master=`cdb2sql -tabs ${CDB2_OPTIONS} ${DBNAME} default 'exec procedure sys.cmd.send("bdb cluster")' | grep MASTER | cut -f1 -d":" | tr -d '[:space:]'`
+cdb2sql ${CDB2_OPTIONS} ${DBNAME} --host $master "PUT SCHEMACHANGE COMMITSLEEP 1"
+cdb2sql ${CDB2_OPTIONS} ${DBNAME} --host $master "PUT SCHEMACHANGE CONVERTSLEEP 1"
 
-max_iter=40
+max_iter=10
 t=0
 while [ $t -lt ${max_iter} ] ; do
+    cdb2sql ${CDB2_OPTIONS} ${DBNAME} --host $master "exec procedure sys.cmd.send('gofast')"
```

Comment:
Use $CDB2SQL_EXE or whatever is the env var for it?

Reply:
Sure, will fix in an upcoming checkin.